src/sql/statement.rs

```diff
@@ -1211,6 +1212,21 @@ fn handle_create_source(scx: &StatementContext, stmt: Statement) -> Result<Plan,
                         Some(_) => bail!("group_id_prefix must be a string"),
                     };
 
+                    max_ts_batch = match with_options.remove("max_ts_batch") {
```

Comment:
I think we should call this `max_timestamp_batch_size`, since most of the other `WITH` options are very spelled out like this.